lib.sh

```diff
@@ -440,34 +440,45 @@ fi
 # https://certbot.eff.org/docs/using.html#certbot-command-line-options
 generate_cert() {
 uir_hsts=""
+disable_000=""
 if [ -z "$SUBDOMAIN" ]
 then
     uir_hsts="--uir --hsts"
 fi
+if [ -e "$SITES_AVAILABLE"/../sites-enabled/000-default.conf ]
```

Comment:
Why did you add this? Just adds complexity when there's no need imo.

Reply:
Or maybe this does something that I missed?

Reply:
In previous versions，  I simply disabled this default site， and reload apache2. When failure to generate certs，This site has not been re-enabled.

Reply:
We use certain files instead of 000-default.conf: $TLS_CONF and $HTTP_CONF.

Here we do that logic in the test script already: https://github.com/nextcloud/vm/blob/master/lets-encrypt/test-new-config.sh#L15-L24

Maybe move that logic to the function instead?